client/app/scripts/liveblog-bloglist/module.js

```diff
@@ -12,6 +12,7 @@
         ];
         $scope.activeState = isArchivedFilterSelected ? $scope.states[1] : $scope.states[0];
         $scope.creationStep = 'Details';
+        $scope.requestAccessMessage = gettext('Click to request access');
```

Comment:
If you don't need to do any action on this string with javascript, the html file would be probably a better place.